src/rpm/package.rs

```diff
@@ -159,6 +159,22 @@ impl Package {
         Ok(())
     }
 
+    // TODO: this should perhaps modify the header in-place instead?
```

Comment:
This will wipe away all but the basic digest tags, which are really the most important ones (the rest being largely deprecated) but it's not the ideal implementation.  But clearing signatures in-place leaving any non-signature tags intact would be about an order of magnitude more complicated, so I'm not doing that right now.